### PR TITLE
Adds ginko registry entry

### PR DESCRIPTION
**Pull request by SethGower**
## Describe your changes
Adds a registry entry for [`ginko_ls`](https://github.com/Schottkyc137/ginko), which is a Device Tree (DTS) Parser and language server. It's very early stages right now, but it's functional. I will also be working on a PR in `nvim-lspconfig` to have a built in configuration for this, as I am currently using a custom defined one.

## Checklist before requesting a review

- [x] I have successfully tested installation of the package.
- [x] I have successfully tested the package after installation.
I tested with the example that the Ginko readme provides. 
```dts
/dts-v1/;

/ {
    pic@10000000 {
        phandle = <1>;
        interrupt-controller;
        reg = <0x10000000 0x100>;
    }
};
```

In order to configure the LSP, I added this to my nvim configs
```lua
    if not configs.ginko_ls then
        configs.ginko_ls = {
            default_config = {
                cmd = { "ginko_ls" };
                filetypes = { "dts" };
                root_dir = function (_)
                    return vim.fs.dirname(vim.fs.find({ '.git' }, { upward = true })[1]);
                end,
                settings = {};
            };
        }
    end
```
Which showed an error after the `}` for `pic@10000000` node.
## Screenshots
image
Screenshot showing that it is appropriately using the `ginko_ls` binary in the mason path.

### CODE REVIEW

File: packages/ginko_ls/package.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+categories:
+  - LSP
+
+source:
+  id: pkg:github/Schottkyc137/ginko@v0.0.4
+  asset:
+    - target: darwin_arm64
+      file: ginko_ls-aarch64-apple-darwin.zip
+      bin: ginko_ls-aarch64-apple-darwin/bin/ginko_ls
+    - target: linux_x64
```
Comment (williamboman) [inline packages/ginko_ls/package.yaml:20]:
```suggestion
    - target: linux_x64_gnu
```

**Review by williamboman — APPROVED** [review]
Thanks!

**Comment by williamboman** [comment]
> FWIW this could also be built from `cargo` so not sure which route is preferable. I figured the released binaries route was better since it's faster, and more reliable with respect to if there is an issue, you can just tell the author of ginko which version is installed from the releases.

Yeah prebuilt release assets are preferred. I saw that the x64 Linux binary is built for GNU/Linux so I updated the target

**Comment by SethGower** [comment]
Awesome, thanks! Any other things you would like addressed?  I realize it's a simple PR, but still. 

Also, just wanted to say great job with this plugin, it's fantastic and makes things _so easy_.

File: packages/ginko_ls/package.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+name: ginko_ls
+description: |
+  ginko_ls is meant to be a feature-complete language server for device-trees. Language servers can be used in many
+  editors, such as Visual Studio Code, Emacs or Vim
+homepage: https://github.com/Schottkyc137
```
Comment (williamboman) [inline packages/ginko_ls/package.yaml:6]:
```suggestion
homepage: https://github.com/Schottkyc137/ginko
```

**Pull request merged**